tests/metrics/test_beyond_label_metrics.py

```diff
@@ -0,0 +1,53 @@
+from tests.common import *
+
+
+def test_cell_cycle(adata_factory):
+    adata = adata_factory()
```

Comment:
any reason why `adata_factory` here, but `adata` below? Here you always re-execute the same function to generate a new `adata` object. It might be worth making `adata_factory` a regular fixture, rather than a factory. Any reason why you want to re-execute that code?

Reply:
I agree. Originally I used `adata_factory` differently, but then I introduced the `adata_pbmc` factory. For cell cycle scores, the PBMC didn't result in enough cell cycle genes, which is why I had to use a different dataset here.

Reply:
I changed this factory to a fixture called `adata_paul15`. Ideally, we would want a single adata fixture which would make this one obsolete. I'd keep it like this for now though.